### ISSUE TITLE
Color is not used as the only visual means of conveying information

### ISSUE THREAD

**Issue by jrpomeroy** [carbon-design-system/carbon#6338]
Moving from https://github.ibm.com/alchemy-containers/argonauts-ui/issues/1244
@xiepingp

===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: Checkpoint 1.4.1 Use of Color
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/color.html
Test Tool Type: Manually Check

===== STEPS TO REPRODUCE =====
Browser: Chrome
Use Visualization Plugin to remove all color from the Web page (Visualization -> ColorBlind -> Luminosity)

Go to https://cloud.ibm.com/ovserve, check all pages with the above configuration.

===== DEFECT DESCRIPTION =====
After removing color from web pages, the hyperlinks (highlighted in red box in attached screenshots) are not clear for user to click.
ColorBlind - Luminosity.zip

===== EXPECTED RESULT =====
The intent of this Success Criterion is to ensure that all users can access information that is conveyed by color differences, that is, by the use of color where each color has a meaning assigned to it.
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color
https://www.w3.org/WAI/WCAG21/Techniques/general/G183.html

![1244 - VCL1](https://media.github.ibm.com/user/9430/files/778bcc00-b56b-11ea-9561-812cdfeaf295)
![1244 - VCL2](https://media.github.ibm.com/user/9430/files/778bcc00-b56b-11ea-8355-9e082121d258)
![1244 - VCL3](https://media.github.ibm.com/user/9430/files/78246280-b56b-11ea-8ef7-a891024a78d5)
![1244 - VCL4](https://media.github.ibm.com/user/9430/files/78246280-b56b-11ea-8595-3916c9a30e9d)
![1244 - VCL5](https://media.github.ibm.com/user/9430/files/78246280-b56b-11ea-87b8-d35b39212ff1)
![1244 - VCL6](https://media.github.ibm.com/user/9430/files/78bcf900-b56b-11ea-9814-c5ce60a2fec5)
![1244 - VCL7](https://media.github.ibm.com/user/9430/files/78bcf900-b56b-11ea-86f9-875815d50188)
![1244 - VCL8](https://media.github.ibm.com/user/9430/files/79558f80-b56b-11ea-851b-1098fef053e1)
![1244 - VCL9](https://media.github.ibm.com/user/9430/files/79558f80-b56b-11ea-890a-0ad47a65b45a)
![1244 - VCL10](https://media.github.ibm.com/user/9430/files/79ee2600-b56b-11ea-9ca0-79b167405417)
![1244 - VCL11](https://media.github.ibm.com/user/9430/files/79ee2600-b56b-11ea-8c68-c9eeb0410180)
![1244 - VCL12](https://media.github.ibm.com/user/9430/files/79ee2600-b56b-11ea-8a85-37cf82bcc7ff)
![1244 - VCL13](https://media.github.ibm.com/user/9430/files/7a86bc80-b56b-11ea-8821-639cf360879c)

**Comment by joshblack** [comment]
Hey there @jrpomeroy! 👋 

For links, if I understand correctly, the requirement for link would fall under: https://www.w3.org/TR/2008/NOTE-WCAG20-TECHS-20081211/G183

Specifically, as long as we provide a visual cue on focus or hover it would pass. I believe we do that for links, does that match your expectation here?

**Comment by joshblack** [comment]
cc @dakahn

**Comment by jrpomeroy** [comment]
@xiepingp thoughts on this?

**Comment by dakahn** [comment]
[This is a misinterpretation of Success Criterion 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html) regarding the Use of Color. [Sufficient Techniques section, situation A, number 4](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G183) has an exception for "...using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them"

Let me know if I'm missing something and I'll reopen 👍

**Comment by xiepingp** [comment]
@jrpomeroy, thank you for raising it for me, yes it makes sense to me!
Thanks @dakahn and @joshblack for your detailed explanation 😀.
